packages/node_modules/@webex/plugin-logger/src/logger.js

```diff
@@ -47,7 +47,7 @@ const authTokenKeyPattern = /[Aa]uthorization/;
 function walkAndFilter(object, visited = []) {
   if (visited.includes(object)) {
     // Prevent circular references
-    return object;
+    return '[Circular]';
```

Comment:
Updated this to return a string. This is due to how loggers in the new Node version do not map this to a string by default when circular.